src/app/public/src/modules/wrapper/wrapper.component.ts

```diff
@@ -66,16 +65,18 @@ export class StacheWrapperComponent implements OnInit, OnDestroy, AfterViewInit
     private navService: StacheNavService,
     private anchorService: StachePageAnchorService,
     private cdr: ChangeDetectorRef,
+    private windowRef: StacheWindowRef,
     private omnibarService: StacheOmnibarAdapterService) { }
 
   public ngOnInit(): void {
     this.omnibarService.checkForOmnibar();
-    this.titleService.setTitle(this.windowTitle || this.pageTitle);
     this.jsonData = this.dataService.getAll();
     this.registerPageAnchors();
   }
 
   public ngAfterViewInit() {
+    this.checkForTutorialHeader();
```

Comment:
I like this idea, but perhaps we can clean up this code some by calling another method.  Maybe a `getTitle()` method that makes the checks and returns the preferred value?  This would clean up the variables being passed into that `setTitle` method.

Reply:
I like that, @Blackbaud-BrandonJones